lib/vdsm/storage/multipath.py

```diff
@@ -209,6 +209,14 @@ def is_ready():
     return "busy: false" in status
 
 
+def reconfigure():
+    """
+    Invoke multipathd to reconfigure the multipaths.
+    Must run as root.
+    """
+    commands.run([_MULTIPATHD.cmd, "reconfigure"])
```

Comment:
We can implement this like resize_map(), so it works also when called as unprivileged user, but we don't have a use case for this so lets keep it simple as is.